src/menu/settings_menu/toggle_music_in_scene.c

```diff
@@ -0,0 +1,62 @@
+/*
+** EPITECH PROJECT, 2022
+** rpg
+** File description:
+** toggle music in scene
+*/
+
+#include "my_rpg.h"
+#include "audio.h"
+
+static int toggle_audio(object_t *obj)
+{
+    sfSoundStatus status = {0};
+
+    if (obj->type != AUDIO) {
```

Comment:
check obj == NULL

Reply:
the loop for check this ptr before this static function are call